README.md

```diff
@@ -40,6 +40,54 @@ mustache.js is a mature project, but it continues to actively invite maintainers
 
 * * *
 
+## Install
+
+Mustache can being easily installed, with npm:
+
+```bash
+$ npm install -g mustache
```

Comment:
Could we recommend a local install, rather than global?

IMO the CLI is the only use case where global install would be recommended, but I'm guessing there's a very small percentage of mustache users who uses that feature.